.appveyor.yml

```diff
@@ -62,6 +62,7 @@ install:
   - echo %PYTHON_VERSION% %TARGET_ARCH%
   # Install dependencies from PyPI.
   - python -m pip install --upgrade -r requirements/testing/all.txt %EXTRAREQS% %PINNEDVERS%
+  - python -m pip install "pybind11<2.10"
```

Comment:
why the pin?

Reply:
I don't recall the precise reason, presumably I was experimenting when the build didn't work. But it doesn't matter now, since merging of https://github.com/matplotlib/matplotlib/pull/24102 we don't need to preinstall `pybind11` for editable installs so changes like this can be reverted.